src/Misc/MiddleWare.cpp

```diff
@@ -298,14 +298,49 @@ struct NonRtObjStore
         return objmap[loc];
     }
 
-    void handleOscil(const char *msg, rtosc::RtData &d) {
-        string obj_rl(d.message, msg);
+    //! try to dispatch a message at the OscilGen non-RT-ports
+    void handleOscilADnote(const char *msg, rtosc::RtData &d, bool voiceOscil) {
```

Comment:
Why would a modulation source be any different from an oscillator getting modulated?
I would have thought for consistency both would be treated more or less identically.